### ISSUE TITLE
VictoryChart, VictoryZoomContainer, VictoryBar Layout Error (iOS)

### ISSUE THREAD

**Issue by yvonne6344** [FormidableLabs/victory-native#193]
Hello, I have a VictoryBar question.

Youtube: [My question 1 video](https://youtu.be/3ZZT0UA825E)
Youtube: [My question 2 video](https://youtu.be/QxuSBJ8WfWk)

I refer  to [this example](https://jsfiddle.net/medmunds/m28x72dz/2/), but VictoryBar x-axis layout error...

<a><img src="https://i.imgur.com/W5DIz3d.png" title="source: imgur.com" width="300" height="100" /></a>

my code:
```
<View width={370}>
        <VictoryChart theme={VictoryTheme.material} domainPadding={15} height={200} width={430}
          containerComponent={
            <VictoryZoomContainer
              dimension="x"
              zoomDomain={{ x: [0.5, 8] }}
              allowZoom={false}
              allowPan={true}
            />}>

          <VictoryBar
            padding={{left:200}}
            style={{ data: { fill: "lightcoral" } }}
            animate={{ duration: 1500 }}
            categories={{ x: ["11/27", "11/28", "11/29", "11/30", "12/1", "12/2", "12/3", "12/4", "12/5"] }}

            data={[
              { x: 1, y: 300, width: 5 },
              { x: 2, y: 700, width: 5 },
              { x: 3, y: 500, width: 5 },
              { x: 4, y: 900, width: 5 },
              { x: 5, y: 700, width: 5 },
              { x: 6, y: 200, width: 5 },
              { x: 7, y: 600, width: 5 },
              { x: 8, y: 300, width: 5 },
              { x: 9, y: 500, width: 5 }
            ]}
          />
        </VictoryChart>
      </View>
```
my version:
```
"react": "16.0.0",
"react-native": "0.50.4",
"react-native-svg": "^6.0.0",
"victory-native": "^0.16.1"
```

**Comment by yvonne6344** [comment]
I try victory-native": "0.15.0, it's success!!
Can fix it in version 0.16.x?
Thanks!!

Youtube: [Zoom VictoryBar Success video](https://youtu.be/N01j3PTWERM)

```
"react": "16.0.0",
"react-native": "0.50.4",
"react-native-svg": "^6.0.0",
"victory-native": "0.15.0"
```

**Comment by boygirl** [comment]
@yvonne6344 It looks like an issue with `VictoryClipContainer` I'll see if I can reproduce this and get it fixed for you

**Comment by boygirl** [comment]
@yvonne6344 Fixed in `victory-native@0.16.3`

**Comment by yvonne6344** [comment]
I use victory-native@0.16.3.
bar & x axis fixed!! but find another issue.

<a href="https://imgur.com/JHKUJAQ"><img src="https://i.imgur.com/JHKUJAQ.png" title="source: imgur.com" width='250' height='150' /></a>
